streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -214,6 +215,20 @@ public RocksDBMetricsRecordingTrigger rocksDBMetricsRecordingTrigger() {
         }
     }
 
+    public final Sensor clientLevelSensor(final String sensorName,
+                                          final RecordingLevel recordingLevel,
+                                          final Sensor... parents) {
+        final String key = CLIENT_LEVEL_GROUP;
+        synchronized (clientLevelSensors) {
+            final String fullSensorName = key + SENSOR_NAME_DELIMITER + sensorName;
```

Comment:
You can inline the value of variable `key` here and remove `key`.

```suggestion
            final String fullSensorName = CLIENT_LEVEL_GROUP + SENSOR_NAME_DELIMITER + sensorName;
```